references/detection/coco_utils.py

```diff
@@ -100,6 +83,27 @@ def __call__(self, image, target):
         return image, target
 
 
+class WrapIntoFeatures:
+    def __call__(self, sample):
+        image, target = sample
+
+        wrapped_target = dict(
+            boxes=features.BoundingBox(
+                target["boxes"],
+                format=features.BoundingBoxFormat.XYXY,
+                image_size=(image.height, image.width),
+            ),
+            # TODO: add categories
+            labels=features.Label(target["labels"], categories=None),
+            masks=features.SegmentationMask(target["masks"]),
+            image_id=int(target["image_id"]),
```

Comment:
This wont work with:
https://github.com/pytorch/vision/blob/59ec1dfd550652a493cb99d5704dcddae832a204/references/detection/engine.py#L27-L29

and leads to:
Unfortunately using the scripts leads to errors:
```
  File "./vision/references/detection/train.py", line 283, in main
    train_one_epoch(model, optimizer, data_loader, device, epoch, args.print_freq, scaler)
  File "./vision/references/detection/engine.py", line 29, in train_one_epoch
    targets = [{k: v.to(device) for k, v in t.items()} for t in targets]
  File "./vision/references/detection/engine.py", line 29, in <listcomp>
    targets = [{k: v.to(device) for k, v in t.items()} for t in targets]
  File "./vision/references/detection/engine.py", line 29, in <dictcomp>
    targets = [{k: v.to(device) for k, v in t.items()} for t in targets]
AttributeError: 'int' object has no attribute 'to'
```